workers/unity/Assets/Playground/Scripts/MonoBehaviours/CubeSpawnerInputBehaviour.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System.Collections.Generic;
+using Generated.Playground;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Core.GameObjectRepresentation;
+using Improbable.Worker;
+using Improbable.Worker.Core;
+using UnityEngine;
+
+#region Diagnostic control
+
+// Disable the "variable is never assigned" for injected fields.
+#pragma warning disable 649
+
+#endregion
+
+namespace Playground.MonoBehaviours
+{
+    public class CubeSpawnerInputBehaviour : MonoBehaviour
+    {
+        [Require] private PlayerInput.Requirables.Reader playerInputReader;
+        [Require] private CubeSpawner.Requirables.Reader cubeSpawnerReader;
+        [Require] private CubeSpawner.Requirables.CommandRequestSender cubeSpawnerCommandSender;
+        [Require] private CubeSpawner.Requirables.CommandResponseHandler cubeSpawnerCommandResponseHandler;
+
+        private ILogDispatcher logDispatcher;
+        private EntityId ownEntityId;
+
+        public void OnEnable()
```

Comment:
Why is this public? Especially given that `Update()` is private. ;)

Reply:
Habit/oversight

Reply:
Fixing